### PR TITLE
Replace dapper chapter with adonet

### LINKED ISSUE #57: Switch orientation Dapper chapter to ADO.NET
We won't introduce Dapper until the API sprint

### PR DESCRIPTION
**Pull request by askingalot**
This may still be a bit rough, feedback is very welcome.

Resolves #57

### CODE REVIEW

File: book-1-orientation/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This book builds a strong foundation in the syntax, data structures, and valuabl
 * Chapter 16 - [Debugging in Visual Studio](./chapters/DEBUGGING_VS.md)
 * Chapter 17 - [Introduction to SQL](./chapters/SQL_INTRO.md)
 * Chapter 18 - [Student Exercises - Part 3](./chapters/STUDENT_EXERCISES_SQL.md)
-* Chapter 19 - [Object Relational Mapping (ORM) with Dapper](./chapters/DAPPER_INTRO.md)
+* Chapter 19 - [Using SQL in a .NET Application](./chapters/ADONET_INTRO.md)
```
Comment (stevebrownlee) [inline book-1-orientation/README.md]: Recommend the following link text

Using ADO.NET and SQL

Putting the concept covered in the title helps students find it faster

**Comment by copypastedeveloper** [comment]
When I talk through ado.net I give them the history of the thing.  I always feel like giving them the context of why it exists is helpful.  I also walk them through all of the different kinds of data access tools we'll cover in class, ado.net, micro-orms, and orms.  Compare and contrast, and discuss how under the covers everyone is using ado.net, and that's why we're learning it.

**Comment by askingalot** [comment]
@copypastedeveloper Do you think we should add anything to the chapter to give that context you mentioned?

@stevebrownlee I made the changes you suggested, would you mind reviewing my the comments I added?

**Comment by stevebrownlee** [comment]
You get a :shipit: from me. @copypastedeveloper want any specific content added that you think would be beneficial?

**Comment by copypastedeveloper** [comment]
I might make a comparison table between those three categories, I drew one on my surface last class, I think I've got it saved somewhere, but probably my handwriting is terrible.  Other than that just talking is probably enough.

**Comment by askingalot** [comment]
I decided to take a stab at a comparison table. I'm not very happy with it. For example, coming up with a con for a micro orm was hard. I imagine that's my bias showing though :)

Any suggestions? Should I take it out? Should I merge it and let one of you take a pass at it?

**Comment by copypastedeveloper** [comment]
yeah, its tough to have negatives for dapper/micro orm's other than I think i had something about doesn't manage connections.

**Comment by copypastedeveloper** [comment]
found my table, its very rough, and not structured like yours.  Yours is much more what i had in mind to make, so if nothing else its a very good starting point.

image

**Comment by askingalot** [comment]
I think I'll just merge what I've got and we'll work from there. After I teach this chapter, I'm sure I'll have a better idea about what changes should be made.

**Pull request merged**